build/build-bundle.js

```diff
@@ -38,8 +39,7 @@ const isDevtools = file => path.basename(file).includes('devtools');
 /** @param {string} file */
 const isExtension = file => path.basename(file).includes('extension');
 
-const BANNER = `// lighthouse, browserified. ${VERSION} (${COMMIT_HASH})\n`;
-const DEBUG = false; // true for sourcemaps
+const FOOTER = `// lighthouse, browserified. ${VERSION} (${COMMIT_HASH})\n`;
```

Comment:
i really like having this version + hash at the top of the file. can we keep it there?

https://www.npmjs.com/package/browserify-banner

Reply:
I don't recall which plugin I tried, but w/e it was did not properly integrate with source maps. I'll try that one.

Reply:
done

![image](https://user-images.githubusercontent.com/4071474/66622271-ec64d480-eb9b-11e9-8d46-0c31fd3709dc.png)

Reply:
max_line_len of 1000 plz

Reply:
oops forgot. ok, u can have your 800 bytes of newlines